services/121-service/src/programs/program/program.service.ts

```diff
@@ -500,30 +503,20 @@ export class ProgramService {
       throw new HttpException({ errors }, 404);
     }
 
-    const connections = await this.connectionRepository
-      .createQueryBuilder('table')
-      .where('1 =1')
-      .getMany();
+    const includedConnections = await this.getIncludedConnections(programId);
 
-    const includedConnections = [];
-    for (let connection of connections) {
-      if (connection.programsIncluded.includes(programId)) {
-        includedConnections.push(connection);
-      }
-    }
-    if (includedConnections.length < 1){
+    if (includedConnections.length < 1) {
       const errors = 'There are no included PA for this program';
       throw new HttpException({ errors }, 404);
```

Comment:
"0 included" could be a valid response, right? At least not a 404... (see below)

Reply:
This part is in the payout function. So if there are 0 included nothing would happen if you press payout.